scripts/setup/create_multinode.sh

```diff
@@ -125,27 +130,25 @@ function setup_workers() {
         node=$1
 
         echo "Setting up worker node: $node"
-        
-        server_exec $node "pushd ~/vhive/scripts > /dev/null && ./setup_tool setup_worker_kubelet ${OPERATION_MODE} && popd > /dev/null"
+
+        server_exec $node "source .profile && cd vhive; ./setup_tool setup_worker_kubelet $OPERATION_MODE"
 
         if [ "$OPERATION_MODE" = "firecracker" ]; then
             setup_vhive_firecracker_daemon $node
         fi
 
-        server_exec $node "sudo ${LOGIN_TOKEN}"
+        server_exec $node "sudo kubeadm join ${MASTER_IP}:${MASTER_PORT} --token ${MASTER_TOKEN} --discovery-token-ca-cert-hash ${MASTER_TOKEN_HASH}"
```

Comment:
That was exactly what was done by the old implementation. Why do we need to change it?